plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -53,6 +59,9 @@ public class ThriftMetastoreConfig
     private int writeStatisticsThreads = 20;
     private boolean batchMetadataFetchEnabled = true;
 
+    private String httpBearerToken;
```

Comment:
What about other authentication in the future? We usually introduce enum property (like `elasticsearch.security` in Elasticsearh connector) for authentication. When we add a new authentication in the future, we may need to use bearer as the default authentication to keep backward compatibility.

Reply:
I can add one but I felt it would be weird to just have one enum value since we only support one authentication mechanism? I feel that it may be more appropriate to use a enum when we add more authentication mechanisms in the future.

Reply:
Let's add please the enum even if it has only one value.